client/dashboard/domains/dataviews/auto-renew-modal.tsx

```diff
@@ -0,0 +1,65 @@
+import { bulkDomainsActionMutation } from '@automattic/api-queries';
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	Button,
+	ToggleControl,
+} from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { _n, __, sprintf } from '@wordpress/i18n';
+import { useState } from 'react';
+import type { DomainSummary } from '@automattic/api-core';
+
+interface AutoRenewModalProps {
+	items: DomainSummary[];
+	onSuccess(): void;
+}
+
+export const AutoRenewModal = ( { items, onSuccess }: AutoRenewModalProps ) => {
```

Comment:
This modal is pending design review. Screenshots attached to the PR description.

Reply:
Commented about it here: p1763076132915079/1763074986.724159-slack-C04H4NY6STW